src/components/index.ts

```diff
@@ -7,6 +7,7 @@ export * from './icon';
 export * from './illustrations';
 export * from './input';
 export * from './progress';
+export * from './radioGroup';
```

Comment:
Even though a `radio` component won't be used on its own, I would structure these components as:
```
/radioGroup
  /radio
    radio.tsx
    [...]
  /radioGroup
    radioGroup.tsx
    [...]
  /radioCard
    radioCard.tsx
    [...]
```
Just to be consistent with the structure of the other components.